src/populate_neo.py

```diff
@@ -29,11 +28,12 @@ def main():
             from_address=item['fromAddress'],
             to_address=item['toAddress'],
             transaction_id=item['id'],
-            nft_id=item['nft']['id']
+            amount=item['amount'],
+            nft = item['nft']
         )
         counter += 1
         transactions.append(transaction)
-        if counter == 2000:
+        if counter == 1000:
```

Comment:
We could let user choose what size does he want

Reply:
✅